optimum/habana/trl/trainer/reward_trainer.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, Union
+
+import torch.nn as nn
+from transformers import (
+    PreTrainedTokenizerBase,
+)
+from transformers.utils import PaddingStrategy
+
+from optimum.habana import GaudiTrainer
+
+
+class GaudiRewardTrainer(GaudiTrainer):
```

Comment:
Can you add a docstring saying that this code comes from https://github.com/huggingface/trl/blob/v0.7.6/examples/research_projects/stack_llama/scripts/reward_modeling.py and not from https://github.com/huggingface/trl/blob/v0.7.6/trl/trainer/reward_trainer.py please?